doc/en/user/source/introduction/gettinginvolved.rst

```diff
@@ -1,52 +1,66 @@
+GeoServer maintains two email lists:
+
+* `GeoServer Users <http://lists.sourceforge.net/lists/listinfo/geoserver-users>`_
+* `GeoServer Developers <http://lists.sourceforge.net/lists/listinfo/geoserver-devel>`_
+
+The Users list is mainly for those who have questions relating to the use of GeoServer, and the Developers list is for more code-specific and roadmap-based discussions. If you see a question asked on these lists that you know the answer to, please respond!
+
+These lists are publicly available and are a great resource for those who are new to GeoServer, who need a question answered, or who are interested in contributing code. 
 
 IRC
 ---
 
-GeoServer has an IRC channel, `#geoserver <irc://irc.freenode.net/geoserver>`_, on the `Freenode <http://freenode.net>`_ network. GeoServer developers frequent this channel, and so it is a great way to give and receive information in real time. 
+GeoServer has an IRC channel, `#geoserver <irc://irc.freenode.net/geoserver>`_, on the `Freenode <http://freenode.net>`_ network. GeoServer developers frequent this channel, and so it is a great way to give and receive information in real time.
 
 Bug tracking
 ------------
 
-If you have a problem when working with GeoServer, then please let us know through the mailing lists. GeoServer uses `JIRA 
-<http://jira.codehaus.org/browse/GEOS>`_ , a bug tracking website, to manage issue reports; you'll need to `create an account first <https://xircles.codehaus.org/>`_. As GeoServer is open source, everyone is encouraged to fix bugs and submit patches. Even if you are not a core developer, we welcome patches through JIRA, or pull requests to `github <https://www.github.com/geoserver/>`_.
+If you have a problem when working with GeoServer, then please let us know through the mailing lists. GeoServer uses `JIRA <http://jira.codehaus.org/browse/GEOS>`_ , a bug tracking website, to manage issue reports. In order to submit an issue, you'll need to `create an account first <https://xircles.codehaus.org/>`_.
+
+Everyone is encouraged to submit patches and, if possible, fix issues as well. We welcome patches through JIRA, or pull requests to GitHub.
 
 Translation
 -----------
 
-We would like GeoServer available in as many languages as possible, just as we want spatial data to be available to all. The two areas of GeoServer to translate are the text for the :ref:`web_admin` and this documentation. Eventually we would even like to set up GeoServer community sites in different languages.  If you are interested in this please let us know via the mailing lists. 
+We would like GeoServer available in as many languages as possible. The two areas of GeoServer to translate are the text that appears in the :ref:`web_admin` and this documentation. If you are interested in helping with this task, please let us know via the mailing lists.
```

Comment:
We have a page on this here http://docs.geoserver.org/latest/en/developer/translation.html but it is outdated.

The blog post here http://blog.geoserver.org/2013/09/03/geoserver-translation-interview/ is more useful.

@fgdrf could you help update the transliteration page?